dashboard/src/components/DeploymentForm/DeploymentFormBody/BasicDeploymentForm/TabularSchemaEditorTable/TabularSchemaEditorTableRenderer.tsx

```diff
@@ -0,0 +1,222 @@
+// Copyright 2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CdsButton } from "@cds/react/button";
+import { CdsIcon } from "@cds/react/icon";
+import ReactTooltip from "react-tooltip";
+import { IBasicFormParam } from "shared/types";
+import ArrayParam from "./Params/ArrayParam";
+import BooleanParam from "./Params/BooleanParam";
+import CustomFormComponentLoader from "./Params/CustomFormParam";
+import SliderParam from "./Params/SliderParam";
+import TextParam from "./Params/TextParam";
+
+const MAX_LENGTH = 60;
+
+function renderCellWithTooltip(
+  value: IBasicFormParam,
+  property: string,
+  className = "",
+  trimFromBeginning = false,
+  maxLength = MAX_LENGTH,
+) {
+  // If the value is an object/array, we need to stringify it
+  const stringValue = ["string", "number"].includes(typeof value?.[property])
+    ? value?.[property] || ""
+    : JSON.stringify(value?.[property]);
+
+  if (stringValue?.length > maxLength) {
```

Comment:
Trimming the string to prevent the cell from overflowing its size: we just show a tooltip with the whole content.